content/docs/manifest.json

```diff
@@ -671,38 +662,48 @@
             }
           ]
         },
-        {
-          "title": "CLI reference",
-          "routes": [
-            {
-              "title": "Introduction",
-              "path": "/docs/cli/README.md"
-            },
-            {
-              "title": "acmesolver",
-              "path": "/docs/cli/acmesolver.md"
-            },
-            {
-              "title": "cainjector",
-              "path": "/docs/cli/cainjector.md"
-            },
-            {
-              "title": "cmctl",
-              "path": "/docs/cli/cmctl.md"
+          {
+            "title": "Reference",
+            "routes": [
+                {
+                "title": "TLS Terminology",
+                "path": "/docs/reference/tls-terminology.md"
             },
-            {
-              "title": "controller",
-              "path": "/docs/cli/controller.md"
+
+                {
+                "title": "Components",
```

Comment:
We've changed the name of this section from "CLI Reference"  to "Components",
because to us "CLI" implies "cmctl: The command line interface to cert-manager"

In another PR we plan to add a CLI top-level heading explaining how to install cmctl and how to use it.